### DISCUSSION TITLE
Help with the inflation!

### DISCUSSION THREAD

**Discussion by zisisp** [denbkh/stoneshard-save-editor#9] [Q&A]
First I wanted to say thank you for releasing this editor.
It's great!
What I want to do is edit the file myself and I have been looking at your code.
Unfortunatly I'm not that good in C# but I wrote the inflater and deflater in Java
I have added the md5 hash but the file generated was not correct. 
Stoneshard sees it as corrupted and now I can't open it anymore. 
Anyway could you do a time edit so that we can fix the failed quests? (If this works I don't know)
For example I'm thinking of going back 3 days so that the 4 day quest is not failed.
Thanks for the code and the tool again

**Comment by zisisp** [comment]
I read the file. Changed the parts in the JSON regarding the time and added MD5 at the end of the file. 
But it still says corrupted.
What I wanted to do first was to compile your code and make the date/time editable and saveable but couldn't make this to run on VS Code :/

**Comment by denbkh** [comment]
> Anyway could you do a time edit so that we can fix the failed quests?

Hi, right now I don't have plans for this feature, sorry.
